scripts/wdl/case_gatk_acnv_workflow.wdl

```diff
@@ -41,54 +53,74 @@ workflow case_gatk_acnv_workflow {
     String seg_param_undoSplits
     Float seg_param_undoPrune
     Int seg_param_undoSD
-    String is_disable_reference_validation
-    String jni_lib    
+
+    # Workflow output directories and options
     String plots_dir
     String call_cnloh_dir
+    String is_disable_reference_validation
     Boolean enable_gc_correction
-
+    Boolean isWGS
+    Int wgsBinSize
 
   call PadTargets {
     input:
         target_bed=target_bed,
         gatk_jar=gatk_jar,
+        isWGS=isWGS,
         mem=1
   }
 
-
   scatter (row in bam_list_array) {
 
-    call AnnotateTargets {
+    call CalculateTargetCoverage as TumorCalculateTargetCoverage {
       input:
           entity_id=row[0],
-          gatk_jar=gatk_jar,
           padded_target_bed=PadTargets.padded_target_bed,
+          input_bam=row[1],
+          bam_idx=row[2],
           ref_fasta=ref_fasta,
           ref_fasta_fai=ref_fasta_fai,
           ref_fasta_dict=ref_fasta_dict,
-          enable_gc_correction=enable_gc_correction,
-          mem=4
-    }
+          gatk_jar=gatk_jar,
+          is_disable_reference_validation=is_disable_reference_validation,
```

Comment:
I don't know if we want the WDL arguments to exactly mirror the CLI arguments, but that's something to consider here in the naming of `is_disable_reference_validation`.  Up to you!